crates/compiler/builtins/bitcode/build.zig

```diff
@@ -83,23 +99,22 @@ fn generateLlvmIrFile(
 fn generateObjectFile(
     b: *Build,
     mode: std.builtin.Mode,
-    target: CrossTarget,
+    target: std.Build.ResolvedTarget,
     main_path: LazyPath,
     step_name: []const u8,
     object_name: []const u8,
 ) void {
-    const obj = b.addObject(.{ .name = object_name, .root_source_file = main_path, .optimize = mode, .target = target, .use_llvm = true });
-    obj.strip = true;
+    const obj = b.addObject(.{ .strip = true, .pic = true, .name = object_name, .root_source_file = main_path, .optimize = mode, .target = target, .use_llvm = true });
+
     obj.link_function_sections = true;
-    obj.force_pic = true;
-    obj.disable_stack_probing = true;
```

Comment:
There are two places where we had this. I removed it based on this comment https://github.com/ziglang/zig/pull/2526#issue-446168498

But I am not sure if that is correct.

Reply:
@bhansconnect maybe this is our issue?

Reply:
Thank you @ostcar for the fix here.